### DISCUSSION TITLE
Compatibility with macOS 13 Ventura?

### DISCUSSION THREAD

**Discussion by spartanhooah** [pilotmoon/Scroll-Reverser#157] [Q&A]
I am responsible for maintaining Scroll Reverser at my company. I do not have a personal Mac to test out compatibility with Ventura. Has anyone tested this yet / does anyone have any reason to suspect any issues?

**Comment by pilotmoon** [comment]
As far as I am aware there are no Ventura-sepcific issues. It works for me on Ventura in my own tests, and I haven't had any reports of issues with Ventura.

**Comment by spartanhooah** [comment]
Thanks!
